feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="overview-price">
+  <ng-container [ngSwitch]="getOverviewPriceType()">
+    <ng-container *ngSwitchCase="overviewPriceType.PRICE_ONLY"
+      ><span>{{ calculateTotal() }}</span></ng-container
+    >
+
+    <ng-container *ngSwitchCase="overviewPriceType.QUANTITY_ONLY"
+      ><span>{{ quantity }}</span></ng-container
+    >
+
+    <ng-container *ngSwitchCase="overviewPriceType.PRICE_AND_QUANTITY"
+      ><span
+        >{{ quantity }} {{ productPrice }} {{ calculateTotal() }}</span
```

Comment:
This is not final, right?
We should not display those numbers without any separators or explanations?

Reply:
Not a part of this ticket. It clearly states, that here we only should add the values, without any styling whatsoever.